src/Kmd.Logic.CitizenDocuments.Client/Kmd.Logic.CitizenDocuments.Client.csproj

```diff
@@ -4,10 +4,12 @@
     <OutputType>Library</OutputType>
     <TargetFramework>netstandard2.0</TargetFramework>
     <RootNamespace>Kmd.Logic.CitizenDocuments.Client</RootNamespace>
+    <PackageIcon>kmd-logic-icon.png</PackageIcon>
```

Comment:
Would it best to put this out the `Directory.Build.props` file with the other common shared nuget properties?

Reply:
Fixed